### ISSUE TITLE
Learn more link in onboarding doesn't teach me more

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot-containers#59]
Found in e7e4537

```sh
$ git rev-parse --short HEAD # e7e4537
```

Clicking the [Learn more](https://github.com/mozilla/testpilot-containers/blob/679d8b9fbb0f3ebceb7448f1ef5b979609264919/webextension/popup.html#L15) link in the popup doesn't seem to open any new tabs or show me any info.

![nightly_and_1____node_modules__bin_jpm_run_-b_nightly__plugin-container__and__eslintrc_js_ _testpilot-containers](https://cloud.githubusercontent.com/assets/557895/21789399/98c57a7a-d689-11e6-96cc-29e728664cfd.png)

about:preferences#privacy Also has a "Learn more" link which seems to take me to https://support.mozilla.org/en-US/kb/containers-experiment?as=u&utm_source=inproduct

![preferences](https://cloud.githubusercontent.com/assets/557895/21789498/6032afb0-d68a-11e6-97b4-5533029518ec.png)

**Comment by ChenMorpheus** [comment]
@pdehaan 
It would be perfect to have someone to help create a specific SUMO page, but it might take too much effort for a Test Pilot experiment. So we defined that clicking the link directs users to the detail page of Containers in Test Pilot in stead. Please find the spec here: https://mozilla.invisionapp.com/d/main#/console/9751862/213055466/preview.

However, if we can revise the SUMO page you mentioned to provide the information for the new UI, we can simply direct to the link. I'll try to find someone for help. Thanks.

**Comment by bakulf** [comment]
What about if we just open the test-pilot container page? Do we have one?
If not, I guess it's better to have the https://support.mozilla.org/en-US/kb/containers-experiment
or maybe, remove the link completely.

**Comment by ChenMorpheus** [comment]
I supposed the test-pilot container page is the detail page of Containers I mentioned above?

**Comment by johngruen** [comment]
Easy path = get rid of link.
Complex path = update sumo page.

**Comment by TanviHacks** [comment]
Maybe we should create a new sumo page that is focused on the Test Pilot addon, rather than the Nightly implementation.
https://support.mozilla.org/en-US/kb/containers-experiment?as=u&utm_source=inproduct

**Comment by TanviHacks** [comment]
I've filed https://bugzilla.mozilla.org/show_bug.cgi?id=1334013 to get a sumo page for containers-testpilot.  I'll work with Joni to populate it once the addon is complete.  Perhaps @kjozwiak can help too.

**Comment by kjozwiak** [comment]
> Perhaps @kjozwiak can help too.

@TanviHacks,  sounds good! Let me know what I can help out with.. We'll probably have to wait till the UI is more polished/finalized if we're going to be adding screenshots into the SUMO page.

**Comment by TanviHacks** [comment]
Not sure if this should be closed as Kamil is still following up on this
for a stretch milestone.

**Comment by pdehaan** [comment]
Reopening this... I think the associated PR removed the Learn More link in the onbearding experience.

**Comment by kjozwiak** [comment]
I've emailed/needinfo'd the SUMO team several times but I haven't received any updates. I'll move this to Sprint2 for now. If I don't hear anything back by the middle of next week, I'm going to close this off.

**Comment by kjozwiak** [comment]
Resolved/WONTFIX [Bug#1334013](https://bugzilla.mozilla.org/show_bug.cgi?id=1334013) as well.
